src/org/opensolaris/opengrok/index/IndexDatabase.java

```diff
@@ -1483,6 +1504,44 @@ private void finishWriting() throws IOException {
         }
     }
 
+    private boolean chkFields(File file, String path) throws IOException {
+        int n = 0;
+        postsIter = uidIter.postings(postsIter);
+        while (postsIter.nextDoc() != DocIdSetIterator.NO_MORE_DOCS) {
+            ++n;
+            // Read a limited-fields version of the document.
+            Document doc = reader.document(postsIter.docID(), CHECK_FIELDS);
+            if (doc == null) {
+                LOGGER.log(Level.FINER, "No Document: {0}", path);
+                continue;
+            }
+
+            /**
```

Comment:
Please use ordinary comments.

Reply:
This kind of review feedback has zero value.